openpype/hosts/webpublisher/plugins/publish/collect_username.py

```diff
@@ -0,0 +1,49 @@
+"""Loads publishing context from json and continues in publish process.
```

Comment:
Move this to ftrack module plugins.

Reply:
I don't like it, its only applicable for 'webpublisher'.

Reply:
Yes that's why `"webpublisher"` is in `hosts`. But the plugin is ftrack specific.

Reply:
Ok, that actually makes sense.

Reply:
BTW instead of using:
```
os.environ["FTRACK_API_USER"] = os.environ["FTRACK_BOT_API_USER"]
os.environ["FTRACK_API_KEY"] = os.environ["FTRACK_BOT_API_KEY"]
```
would be better to just pass api key and username to `Session` arguments
```
ftrack_username = os.environ["FTRACK_BOT_API_USER"]
ftrack_api_key = os.environ["FTRACK_BOT_API_KEY"]

session = ftrack_api.Session(
    api_key=ftrack_api_key,
    api_user=ftrack_username,
    auto_connect_event_hub=False
)
```